### DISCUSSION TITLE
Output of camera bboxes3d in different format for SMOKE and FCOS3D

### DISCUSSION THREAD

**Discussion by turtlemicro** [open-mmlab/mmdetection3d#2956] [Q&A]
Hey,
I am doing inference with my own data on pretrained models, using the MonoDet3DInferencer class. The inference runs, but I get different ouputs for my bounding boxes for the different models. I get 7 values for SMOKE, which is like in the docu explained x, y, z, dx, dy ,dz, yaw angle. So far so good. But for the boundingboxes of FCOS3D I get 7 values. The first 6 values look like they correspond also to x, y, z, dx, dy, dz, but I am not sure about the last three values. Does FCOS3D outputs also the pitch and the role angle? I don't think that makes sense. Examples below.
Thank you for your help in advance.

SMOKE:
{"labels_3d": [0, 2, 2, 0, 2, 2, 0, 0, 2], "scores_3d": [0.4008917212486267, 0.3517446517944336,...], "bboxes_3d": [[2.287151336669922, 0.05406087636947632, 10.526403427124023, 0.9118178486824036, 1.861153483390808, 0.6629536151885986, -0.2253173440694809], [19.52056884765625, -5.382413387298584, 38.825279235839844, 3.6742231845855713, 1.4865235090255737, 1.614647626876831, -1.9082417488098145],...

FCOS3D: 
{"labels_3d": [0, 2, ...], "scores_3d": [0.4564345212486267,...],"bboxes_3d": [[19.078702926635742, 0.45838648080825806, 24.945350646972656, 4.562831878662109, 1.6041855812072754, 1.9172170162200928, 4.62945556640625, 0.04231985658407211, 0.2161380648612976], [7.540063858032227, 0.7213494777679443, 15.121360778808594, 0.7212233543395996, 1.6502604484558105, 0.5703033208847046, 3.7032604217529297, -0.23937293887138367, -0.13246335089206696], ...

**Comment by rlczddl** [comment]
The last two are speed
